src/os/bluestore/BlueStore.cc

```diff
@@ -414,6 +414,23 @@ static int get_key_object(const S& key, ghobject_t *oid)
   return 0;
 }
 
+int BlueStore::objkey_to_oid(const std::string& key, ghobject_t *oid)
+{
+  return get_key_object(key, oid);
+}
+
+int BlueStore::extentkey_to_oid(const std::string& key, ghobject_t *oid, uint32_t *offset)
```

Comment:
using 'extent' here and below is confusing - this object is extent shard - a bunch of extents of an onode which use separate KV entry for the sake of having smaller entries to operate on. Hence suggest either use shard or extent_shard or something